### PR TITLE
Adding Motr Configuration Document

### PR DESCRIPTION
**Pull request by VenkyOS**
Signed-off-by: Venkataraman Padmanabhan <venkataraman.padmanabhan@seagate.com>

### CODE REVIEW

File: doc/Motr_Configuration.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
+==========
```
Comment (huanghua78) [inline doc/Motr_Configuration.rst:1]: Where is the original document? Where is the document referenced?

I don't think we need this document any longer.

Comment (VenkyOS) [reply doc/Motr_Configuration.rst:1]: Link to the document: [Source Document](https://seagatetechnology-my.sharepoint.com/:w:/r/personal/valery_vorotyntsev_seagate_com/_layouts/15/Doc.aspx?sourcedoc=%7BC7B2AC10-5EB5-442E-836E-CE6530382B6E%7D&file=Colibri%20Configuration%201-pager.docx&action=default&mobileredirect=true)

Comment (VenkyOS) [reply doc/Motr_Configuration.rst:1]: If you confirm, I can close this PR.

Comment (huanghua78) [reply doc/Motr_Configuration.rst:1]: Please mark this document as "not needed". It is out-dated, and if it exists, it will be misleading.

**Comment by huanghua78** [comment]
This doc is not needed.

**Pull request closed without merging**